libs/icons/figma-icon-config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Thoughts on adding this file to the `gitignore`? Could help to prevent accidental commits of personal access tokens… which I may or may not have done before 😓

Reply:
That's a good thought and question. We don't want to ignore this file because it is dependent. Access tokens should be in the .env file.